frontend/src/pages/About.jsx

```diff
@@ -1,3 +1,44 @@
+import youngProfessionals from "../images/young-professionals.jpg"
+import "../style/About.css"
+
 export default function About() {
-  return <h1>About Page</h1>
+  return (
+    <>
+      <div className="about-page-container">
```

Comment:
Think this should be a main tag got each page component

Reply:
done